charts/rollups-node/script.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# Script to convert values from rollups node TOML file to helm chart values YAML file with descriptions
+
+# Define file names
+TOML_FILE="Config.toml"
+TEMPLATE_VALUES_FILE="values.yaml.tpl"
+VALUES_FILE="values.yaml"
+
+# Download the Config.toml file from the rollups-node repository
+wget https://raw.githubusercontent.com/cartesi/rollups-node/main/internal/config/generate/Config.toml
+
+# Use the template TOML values file as a starting point
+echo "Using $TEMPLATE_VALUES_FILE as the template values file..."
+cat "$TEMPLATE_VALUES_FILE" > "$VALUES_FILE"
+
+# Process TOML file using yq and transform to YAML format
+echo "Processing $TOML_FILE using yq and transforming it to YAML format..."
+yq -ptoml -oy '.[] | to_entries' "$TOML_FILE" |
+
+# Process each entry, extract description and default value, format as YAML
+yq eval-all '.[] | {"description": .value.description, .key: .value.default}' |
+
+# Remove "description:" key
+sed 's/description: //g' | sed '/^ *|-/d' |
+
+# Make comments for each key value pairs
+awk '/^[[:space:]]*[^#[:space:]]*:/ {print $0; next} {print "#", $0}' |
+
+# Add "--" to the first comment line for auto Readme
+awk '/^ *#/ {if (!comment) sub("#", "# --"); comment=1} /^#/{print; next} {comment=0; print}' |
+
+# Remove "null" values
+sed 's/: null$/: ""/g' |
+
+# Add 4 spaces of indentation to each line and finally add the values to value.yaml
+sed 's/^/    /' >> "$VALUES_FILE" |
```

Comment:
```suggestion
cat Config.toml \
| yq -ptoml -oy '
  .[]
  | to_entries
  | map
    (
      {
        .key: .value.default // "",
        "description": .value.description
      }
    )
  | .[] headComment |= "-- " + .description
  | del(.[].description)
  | .[]
' \
| yq '
  {
    "validator":
      {
        "config": .
      }
  }' \
| sed '/^validator:/d;/^$/d'
```

Since TOML is well structured, we don't wanna trust on regexp with `sed` and `awk` too much, it's safer to use `yq` for the job.

We could even avoid some file manipulation processing the curl stream, something like:

```
curl --proto '=https' --tlsv1.2 -sSf -L https://raw.githubusercontent.com/cartesi/rollups-node/main/internal/config/generate/Config.toml \
| yq -ptoml -oy '
...
```